### ISSUE TITLE
Creation of multiple CUDA Contexts can lead to crash

### ISSUE THREAD

**Issue by leeping** [pandegroup/openmm#45]
In my MD script, sometimes I need to run a short sequence of simulations on different systems.  Specifically, the first one is a condensed phase simulation, and the second one is a gas phase simulation (just a single molecule).  This requires creating multiple Contexts, and I found that creation of the second Context on the CUDA platform can cause the script to crash.

I have seen this problem on a few machines, but not all; the latest such error occurred on Blue Waters.  So far I have been able to work around the problem by using the reference platform for the gas phase simulation, but it is probably worth getting to the bottom of this.

My script is here: https://simtk.org/websvn/wsvn/forcebalance/src/data/npt.py

```
Traceback (most recent call last):
  File "npt.py", line 1200, in <module>
    main()
  File "npt.py", line 976, in main
    mData, mXyzs, _trash, _crap, mPotentials, mKinetics, _nah, _dontneed, mSim, mEDA = run_simulation(mpdb, mSettings, pbc=False, Trajectory=False)
  File "npt.py", line 461, in run_simulation
    simulation, system = create_simulation_object(pdb, settings, pbc, "mixed")
  File "npt.py", line 444, in create_simulation_object
    simulation = Simulation(mod.topology, system, integrator, platform)
  File "/u/sciteam/leeping/local/lib/python2.7/site-packages/simtk/openmm/app/simulation.py", line 77, in __init__
    self.context = mm.Context(system, integrator, platform)
  File "/u/sciteam/leeping/local/lib/python2.7/site-packages/simtk/openmm/openmm.py", line 10903, in __init__
    this = _openmm.new_Context(*args)
Exception: Error initializing Context: CUDA_ERROR_INVALID_DEVICE (101) at /u/sciteam/leeping/src/OpenMM/platforms/cuda/src/CudaContext.cpp:136
```

**Comment by peastman** [comment]
It's been almost a year since John said he thought this was fixed in nvcc 6.0, and no one has commented since then, so I'll go ahead and close it.

**Comment by jchodera** [comment]
We haven't run into the issue since then, thankfully!
